### ISSUE TITLE
How To install this nice sample on MS windows

### ISSUE THREAD

**Issue by gitlove123** [oatpp/oatpp-starter#9]
You do nice work!

I am new of oatpp and cmake. But I have succeeded in installing this sample on Mac OSX.

<img width="656" alt="oatpp-starter-mac" src="https://user-images.githubusercontent.com/67555832/87242494-19665080-c468-11ea-9a2c-0e83f1d5fe51.png">

great!

But when I tried to install it on MS Windows OS, I got the following error message:

<img width="808" alt="oatpp-starter-x86orx64" src="https://user-images.githubusercontent.com/67555832/87242601-30f20900-c469-11ea-9d16-023886dbee98.png">

How to solve it?

**Comment by lganzzzo** [comment]
Hey @gitlove123 ,

From the log message, it looks like you are trying to build `oatpp-starter` project, but the main `oatpp` module is missing.
So it's whether you haven't installed the main `oatpp` module or for some reason, Cmake can't see it.

- Please make sure that the main oatpp module is installed.
- [if oatpp is installed] - Try to find out the reason why CMake can't find the installed module.

Please let me know how it goes!

Regards,
Leonid

**Comment by gitlove123** [comment]
Hi, @lganzzzo 

Thank you for your quick response.

I had built oatpp module prior to installing this project:

![oatpplib-win10](https://user-images.githubusercontent.com/67555832/87245269-0d39bd80-c47f-11ea-89d9-60279ed6c1c9.PNG)

It's OK?

I suspect that cmake on win 10 build it for x86 architecture.

Honestly speaking, it was done at Parallels win 10 on Mac OSX.

I will try it again on NATIVE windows 10 machine.

**Comment by lganzzzo** [comment]
Hey @gitlove123 ,

I double-checked the error message - it says:

```
Could not find ... requested version 1.1.0
...
The following were considered but not accepted:
... version 1.1.0
```

So it actually found the library, but for some reason, it didn't accept it.
Did you build both projects on the same machines?
The versions seem to be identical so CMake should have accepted it.
At the moment I have no idea why CMake didn't accept the lib with the correct version.

Did you build oatpp as described [here](https://oatpp.io/docs/installation/windows/), or did you use some additional commands/configs?

---

Just to clarify:
> I had built oatpp module prior to installing this project:

- The main `oatpp` module is installable.
- The `oatpp-starter` is the starter for a non-installable project. It should be just built and run.

---

Please let me know if you have any progress on this one, or if you have more questions.

Regards,
Leonid

**Comment by gitlove123** [comment]
Hi, @lganzzzo 

I tried again on win10 native machine.
As a result, this sample works well !

![oatpp-starter-2](https://user-images.githubusercontent.com/67555832/87322878-f66f9580-c568-11ea-98ce-1e9452d72603.PNG)

Thanks.

I guess that my wrong point was  I tried to install oatpp module including .lib and header files into C:/usr/local directory as a non-administrator because I dislike C:/Program Files directory.
In this case, I as a admin of my win10 machine installed it correctly.
Then, cmake found them and made .sln and .vcxproj files of this sample project:

![oatpp-starter-1](https://user-images.githubusercontent.com/67555832/87325150-0c328a00-c56c-11ea-958b-15149a30d946.PNG)

Next, I build it using visual studio:

![oat-starter-studio](https://user-images.githubusercontent.com/67555832/87325322-4439cd00-c56c-11ea-9a55-8725f2fc0d95.PNG)

The .exe file works fine.


Thanks.
